omf-python/src/colormap.rs

```diff
@@ -0,0 +1,87 @@
+use crate::array::{PyBoundaryArray, PyGradientArray};
+use omf::{NumberColormap, NumberRange};
+use pyo3::exceptions::PyValueError;
+use pyo3::prelude::*;
+use pyo3_stub_gen::derive::*;
+
+#[gen_stub_pyclass]
+#[pyclass(name = "NumberColormapContinuous")]
+/// A continuous colormap linearly samples a color gradient within a defined range.
+///
+/// A value X% of way between `min` and `max` should use the color from X% way down
+/// gradient. When that X doesn't land directly on a color use the average of
+/// the colors on either side, inverse-weighted by the distance to each.
+///
+/// Values below the minimum use the first color in the gradient array. Values above
+/// the maximum use the last.
+pub struct PyNumberColormapContinuous(pub NumberColormap);
+
+#[gen_stub_pymethods]
+#[pymethods]
+impl PyNumberColormapContinuous {
+    /// Value range.
+    fn range(&self, py: Python<'_>) -> PyResult<PyObject> {
+        match self.0 {
+            NumberColormap::Continuous { range, .. } => match range {
+                NumberRange::Float { min, max, .. } => Ok((min, max).into_py(py)),
+                NumberRange::Integer { min, max, .. } => Ok((min, max).into_py(py)),
+                NumberRange::Date { min, max, .. } => Ok((min, max).into_py(py)),
+                NumberRange::DateTime { min, max, .. } => Ok((min, max).into_py(py)),
+            },
+            _ => Err(PyValueError::new_err(
+                "NumberColormap variant is not supported",
+            )),
```

Comment:
Other similar code is just panicking in this case, which I think is better, as it indicates that reaching here requires a programming error in these bindings. Remaining consistent is better too.
```suggestion
            _ => unreachable!(),
```